lib/widgets/screens/profile/profile_basic_info.dart

```diff
@@ -0,0 +1,231 @@
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:mm_flutter_app/constants/app_constants.dart';
+import 'package:mm_flutter_app/utilities/scaffold_utils/report_or_block_menu_button.dart';
+import 'package:mm_flutter_app/widgets/atoms/profile_chip.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+const double _inlineIconSize = 16.0;
+
+Future<void> _launchUrl(Uri url) async {
+  if (!await launchUrl(url)) {
+    throw Exception('Could not launch $url');
+  }
+}
+
+Widget _createFooter(BuildContext context, String company, String? companyRole,
+    String? education, String? linkedinUrl) {
+  String companyAndRole = company;
+
+  final ThemeData theme = Theme.of(context);
+  if (companyRole != null) {
+    companyAndRole = "$companyRole, $company";
+  }
+
+  return Row(mainAxisAlignment: MainAxisAlignment.spaceBetween, children: [
+    Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
+      Wrap(
+        crossAxisAlignment: WrapCrossAlignment.center,
+        children: [
+          const Icon(
+            Icons.work_outline,
+            size: _inlineIconSize,
+          ),
+          const SizedBox(width: Insets.paddingExtraSmall),
+          Text(
+            companyAndRole,
+            style: theme.textTheme.labelSmall!
+                .copyWith(color: theme.colorScheme.secondary),
+          ),
+        ],
+      ),
+      if (education != null) const SizedBox(height: Insets.paddingExtraSmall),
+      if (education != null)
+        Wrap(
+          crossAxisAlignment: WrapCrossAlignment.center,
+          children: [
+            const Icon(
+              Icons.school_outlined,
+              size: _inlineIconSize,
+            ),
+            const SizedBox(width: Insets.paddingExtraSmall),
+            Text(
+              education,
+              style: theme.textTheme.labelSmall!
+                  .copyWith(color: theme.colorScheme.secondary),
+            ),
+          ],
+        ),
+    ]),
+    if (linkedinUrl != null)
+      Ink(
+        decoration: ShapeDecoration(
+          shape: RoundedRectangleBorder(
+              borderRadius:
+                  BorderRadius.circular(Radii.roundedRectRadiusSmall)),
+          color: theme.colorScheme.onInverseSurface,
+        ),
+        child: IconButton(
+          style: IconButton.styleFrom(
+              shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(Radii.roundedRectRadiusSmall),
+          )),
+          icon: const ColorFiltered(
+            colorFilter: ColorFilter.mode(
+              Colors.white,
+              BlendMode.saturation,
+            ),
+            child: SizedBox(
+              width: _inlineIconSize,
+              child: Image(
+                image: AssetImage(Assets.linkedInIcon),
+              ),
+            ),
+          ),
+          onPressed: () {
+            debugPrint(linkedinUrl);
+            _launchUrl(Uri.parse(linkedinUrl));
```

Comment:
Curious: how does this work when you have the app? Does it open the linkedIn url on the browser or within the app?

Reply:
Great point! Added a comment below